src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -78,7 +78,9 @@ public static unsafe void Ctor_CharPtr_OddAddressShouldStillWork()
         [InlineData(new char[] { '\0' }, 0, 1, "\0")]
         [InlineData(new char[] { 'a', 'b', 'c' }, 0, 0, "")]
         [InlineData(new char[] { 'a', 'b', 'c' }, 1, 0, "")]
+#pragma warning disable xUnit1026 // Unused parameter
```

Comment:
It looks like a bug in the analyser. Suppressed for now.

Reply:
I didn't see a bug report for this; consider filing it at https://github.com/xunit/xunit

Reply:
https://github.com/xunit/xunit/issues/1969